app/src/scenes/Home/components/catogories/interfaces.ts

```diff
@@ -4,15 +4,10 @@ export interface CategoryStateProps {
 
 export interface Category {
   id: number;
-  translations: {
-    en: {
-      name: string,
-    },
-    si?: {
-      name: string,
-    },
-    ta?: {
-      name: string,
-    },
-  };
+  translations: Language[];
+}
+
+export interface Language {
```

Comment:
Let's rename this interface as `CategoryTranslation` 
Reasons: 
- We are supposed to create a language interface later to list available languages. So the developers might be confused
- This translation is unique for a category. SubCategory and Item translations' structure might be different

Reply:
sure thanks for the reasons.